zanata-war/src/main/java/org/zanata/exception/handler/ProjectNotFoundExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015, Red Hat, Inc. and individual contributors as indicated by the
```

Comment:
- [ ] For new files, this should be 2016. (Also VersionNotFoundExceptionHandler.java).